packages/contracts-bedrock/foundry.toml

```diff
@@ -53,8 +53,9 @@ ignore = ['src/vendor/WETH9.sol']
 #                         PROFILE: CI                          #
 ################################################################
 
-[profile.ci.fuzz]
-runs = 512
+[profile.ci]
+deny_warnings = true
+fuzz = { runs = 12 }
```

Comment:
Wait this should be 512 not 12

Reply:
oops I typo'd that in my suggestion [here](https://github.com/ethereum-optimism/optimism/pull/8456#discussion_r1416410591), thanks for catching @tynes